index.html

```diff
@@ -1,71 +1,99 @@
-          <button class="btn btn-primary" id="textToAudio">Generate Audio 1</button>
-          <button class="btn btn-primary" onclick="convertText()" disabled>Generate Audio 2</button>
-        </div>
-      </div>
+                    <div class="d-flex flex-column align-items-center">
+                        <div class="mb-2">
+                            <button
+                                class="btn btn-primary"
+                                id="convertPdfToAudioFromDialog"
+                            >
+                                Choose PDF File and Convert
+                            </button>
+                            <!-- Add "i" icon with tooltip "Click and choose your file from file dialog and will converted automatically" -->
+                        </div>
+                        <div
+                            class="d-flex align-items-center justify-content-center"
+                        >
+                            <audio id="pdfAudioPlayer" controls></audio>
+                            <button
+                                class="btn btn-danger ms-2"
+                                id="deleteAudio"
+                                onclick="alert('This will work soon...')"
+                            >
+                                <i class="bi bi-trash3"></i>
+                            </button>
+                        </div>
+                    </div>
+                </div>
 
-      <!-- create obly button for "pdf-to-audio" handler, the rest will be handled by handler when button clicked -->
-    </div>
-  </main>
-  <footer class="d-flex justify-content-center bg-primary text-white p-1">
-    <p class="text-center mb-0">&copy; 2024 PDF to Audio. All rights reserved.</p>
-  </footer>
+                <div
+                    class="pdf-to-audio-container d-flex flex-column align-items-center mb-4"
+                >
+                    <h2 class="text-primary mb-2">Convert Text to Audio</h2>
+                    <textarea
+                        class="text-to-audio-textarea"
+                        id="textInput"
+                        placeholder="Enter text..."
+                    ></textarea>
+                    <div class="d-flex gap-2 align-items-center mt-2">
+                        <label for="langSelect">Language:</label>
+                        <select id="langSelect">
+                            <option value="en">English</option>
+                            <option value="ru">Russian</option>
+                        </select>
+                    </div>
+                    <div class="d-flex gap-2 mt-2 align-items-center">
+                        <!-- <button class="btn btn-primary" id="textToAudio">Generate Audio 1</button> -->
+                        <button
+                            class="btn btn-primary"
+                            onclick="convertText()"
+                            disabled
```

Comment:
Please remove disabled and bring `id="textToAudio"` to this button.

Forgot to mention on the issue :) this button serves as the main feature :)